sim/rogue/rotation_generic.go

```diff
@@ -374,7 +375,7 @@ func (x *rotation_generic) shouldCastNextRotationItem(sim *core.Simulation, rogu
 
 	// Expires before next GCD
 	if tte <= timeUntilNextGCD {
-		if comboPoints >= prio.MinimumComboPoints && currentEnergy >= (prio.EnergyCost+prio.PoolAmount) {
+		if comboPoints >= prio.MinimumComboPoints && currentEnergy >= (prio.EnergyCost+prio.PoolAmount) && prio.GetSpell(rogue, comboPoints).CanCast(sim, rogue.CurrentTarget) {
```

Comment:
prio.GetSpell() returns `nil` for the MCD enabler, and spell.CanCast() returns `false` in this case - which might end up in an endless `return ShouldWait` (here or elsewhere)